backend/oasst_backend/config.py

```diff
@@ -21,6 +21,7 @@ class Settings(BaseSettings):
     DEBUG_ALLOW_ANY_API_KEY: bool = False
     DEBUG_SKIP_API_KEY_CHECK: bool = False
     DEBUG_USE_SEED_DATA: bool = False
+    DEBUG_USE_SEED_DATA_PATH: FilePath = "backend/test_data/generic/test_generic_data.json"
```

Comment:
```suggestion
    DEBUG_USE_SEED_DATA_PATH: Optional[FilePath] = Path(__file__).parent.parent / "test_data/generic/test_generic_data.json"
```
needs `from pathlib import Path`

Reply:
Thanks. More safe to use pull path.